launcher/src/main/java/org/apache/spark/launcher/SparkAppHandle.java

```diff
@@ -95,7 +95,8 @@ public boolean isFinal() {
   void kill();
 
   /**
-   * Disconnects the handle from the application, without stopping it. After this method is called,
+   * Disconnects the handle from the application. If using {@link SparkLauncher#autoShutdown()}
+   * option, this method would shutdown stop/kill the application. After this method is called,
```

Comment:
Instead of "method would shutdown stop/kill the application", say "this method will also stop the child Spark application".